js-miniapp-sdk/README.md

```diff
@@ -81,12 +81,17 @@ miniApp.requestLocationPermission()
 	});
 ```
 
-### 4. Show Interstitial Ad
+### 4. Show Ads
 
-Mini App JS SDK makes it possible to display Interstitial ads during the execution of a mini-app. MiniApp#showInterstitialAd facilitates the display of the Ad while pairing with the Mini App Native SDKs. The results of the display action are captured as [InterstitialAdResponse](src/types/responseTypes/interstitial/index.ts).
-Note: This requires the mini-app to have obtained/enrolled the required ad units with the host app.
+Mini App SDK allows to display ads upon requesting from Mini apps.
+This requires the mini-app to have obtained/enrolled the required ad units with the host app.
+Currently two ad types are supported,
+1. Interstitial
+2. Rewarded
 
-There is an adType key in the response type which is a value from AdTypes enum and the value for this response is set to `AdTypes.INTERSTITIAL`
+| Interstitial |
+| ------------ |
```

Comment:
This will make this formatted as a table, which looks a little weird as a section header. Maybe just make it bold instead?

```suggestion
**Interstitial**
```

Reply:
Removed table design!